bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,233 @@ class StylintBear:
     SEE_MORE = 'https://github.com/SimenB/stylint'
 
     @staticmethod
+    def generate_config(filename, file,
+                        block_keyword: bool=None,
+                        brackets: bool=False,
+                        colons: bool=True,
+                        color_variable_over_hex_value: bool=True,
+                        spaces_after_commas: bool=True,
+                        spaces_after_comments: bool=True,
+                        allow_trailing_whitespace: bool=False,
+                        css_literal: bool=False,
+                        max_selector_depth: bool=False,
+                        check_duplicates: bool=True,
+                        efficient_properties: bool=True,
+                        extend_preference: str=None,
+                        indent_size: int=0,
+                        leading_zero: bool=False,
+                        max_errors: int=0,
+                        max_warnings: int=0,
+                        mixed_spaces_and_tabs: bool=False,
+                        variable_naming_convention: str=None,
+                        strict_naming_convention: bool=False,
+                        none_keyword: bool=False,
+                        check_no_important_keyword: bool=True,
+                        spaces_inside_parentheses: bool=None,
+                        placeholder: bool=True,
+                        prefix_vars_with_dollar: bool=True,
+                        semicolons: bool=False,
+                        sort_order: str='alphabetical',
+                        stacked_properties: bool=False,
+                        check_valid_property: bool=True,
+                        zero_units: bool=False,
+                        z_index_normalize: bool=False,
+                        stylint_config: str=''):
+        """
+        :param block_keyword:
+            When ``True`` expect the ``@block`` keyword when defining block
+            variables. When ``False``, expect no ``@block`` keyword when
+            defining block variables.
+        :param brackets:
+            When ``True``, expect ``{}`` when declaring a selector. When
+            ``False``, expect no brackets when declaring a selector.
+        :param colons:
+            When ``True``, expect ``:`` when declaring a property. When
+            ``False``, expect no ``:`` when declaring a property.
+        :param color_variable_over_hex_value:
+            When ``True``, enforce variables when defining hex values.
```

Comment:
over -> for? "Over" sounds like it should look like this:
```
red
0xFFFF0000
```
instead of
```
red = 0xFFFF0000
```
(then also append a pluralizing "s" again, sounds a bit more general)